node/cn/tracers/testdata/call_tracer_selfdestruct.json

```diff
@@ -57,7 +57,10 @@
   "result": {
     "calls": [
       {
-        "type": "SELFDESTRUCT"
+        "type": "SELFDESTRUCT",
+        "from": "0xf0b1b2a91af3b0a0a5389ea80bffdc42cf86b7e3",
+        "to": "0xf0b1b2a91af3b0a0a5389ea80bffdc42cf86b7e3",
+        "value": "0x0"
```

Comment:
How about adding another testcase for non-zero value?

Reply:
@jeongkyun-oh 
Great. I will add the test case for non-zero value too :)

Reply:
@jeongkyun-oh 
I just updated the test.
balance of the contract is bigger than zero.
Because most of the scenario is same except for balance, so I chose to update the test case instead of copy and paste + update only balance field. 

Please let me know if we should keep the original test case.